azure-pipelines.yml

```diff
@@ -95,6 +95,8 @@ jobs:
         - group: DotNet-Blob-Feed
         - group: DotNet-Symbol-Server-Pats
       steps:
+      - powershell: ./eng/scripts/InstallVisualStudio.ps1 -Edition BuildTools -Quiet
```

Comment:
Don't let this stop the PR, but I think this confuses things since BuildTools isn't an actual Edition. Maybe change the parameter name to "workload"?

Reply:
BuildTools is both a workload and an edition -- you can download Build Tools for Visual Studio 2019, Visual Studio 2019 Community, et cetera.